### DISCUSSION TITLE
请问在既有动态库又有静态库的情况下，怎么指定能优先链接静态库?

### DISCUSSION THREAD

**Discussion by whwalker** [xmake-io/xmake#3672] [Q&A]
比如我在系统上安装log4cpp后，系统lib目录下有liblog4cpp.dylib和liblog4cpp.a，怎么指定才能优先使用liblog4cpp.a？看了文档没有找到有效的方法

**Comment by whwalker** [comment]
忘了说了，我的使用方法如下：
add_requires("log4cpp", {system=true})
target("Project")
	add_packages("log4cpp")
	set_kind("binary")
        ...
试过用 add_requires("log4cpp", {configs ={shared=false}}) 不启作用

**Reply by waruqi** [reply]
那是因为目前没 log4cpp 这个包，只有 log4cplus包，你可以提个 pr 过来，到 xmake-repo 仓库
